zoo_api/lib/Stored Procedures/GetLastDietChanges.sql

```diff
@@ -0,0 +1,10 @@
+CREATE DEFINER=`zoo`@`%` PROCEDURE `zoo`.`getLastDietChanges`(IN num int(10))
+BEGIN
```

Comment:
nit: I'd like a default value on the SP. `SET num = IFNULL(num, 3);` or whatever default value you want.

Reply:
The procedure cant run without the param even if I set a default like you say.

Reply:
You are correct, but in the loopback API implementation for the SP you can technically send in undefined for `n` and that will come up as NULL. Having the check in the SP makes more sense to me, but if you want to check in the API that's fine too.

Reply:
I really dont think this needs a default value.